cypress/integration/integration_spec.js

```diff
@@ -62,10 +242,10 @@ describe('e2e', function() {
         cy.get('#js-mobile-nav-toggle').as('navToggle');
         cy.get('#qa-offscreen-navigation').as('nav');
 
-        cy.get('@navToggle').click();
+        cy.get('@navToggle').click({ force: true });
```

Comment:
What's this doing (and why)?

Reply:
Oh good point, there was something funny going on with the scroll position when I upgraded Cypress, so this bypasses the visibility check. This was a workaround. It's not a great one though, so let me double check.

Reply:
I don't mind the code, just wasn't clear what it was doing. Add a comment maybe?

Reply:
If I can't find a better solution then will do